<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelEditor</class>
 <widget class="QWidget" name="LabelEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>704</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="toolBar">
     <item>
      <widget class="QToolButton" name="tbtnUndo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbtnRedo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Redo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnResetZoom">
       <property name="text">
        <string>Reset zoom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnZoomBug">
       <property name="text">
        <string>Zoom on bug</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="toolBox">
       <property name="title">
        <string>Mask tools</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="MaskGroup">
       <property name="title">
        <string>Active mask</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="tbtnBugMask">
          <property name="text">
           <string>Bug</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtnSegmentsMask">
          <property name="text">
           <string>Segments</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtnReflectionMask">
          <property name="text">
           <string>Reflections</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnResetLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="center">
     <item>
      <layout class="QVBoxLayout" name="photo_view">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="controls">
         <item>
          <widget class="QPushButton" name="btnPrevious">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hspcLeft">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="hspcRight">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnNext">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabSidebar">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabEditing">
        <attribute name="title">
         <string>Editing</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabPlugins">
        <attribute name="title">
         <string>Plugins</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
